src/__tests__/jest/components/EditProfile.test.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import configureStore from "redux-mock-store";
+import thunk from "redux-thunk";
+import EditProfile from "../../../components/profile/EditProfile";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+const middleware = [thunk];
+const mockStore = configureStore(middleware);
+const store = mockStore({});
+const props = {
+  profile: {
+    profile: {
```

Comment:
is the profile necessary in this test?

Reply:
No, it is not necessary. I have refactored to remove this.